src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.DecryptTokenResult.cs

```diff
@@ -62,8 +50,9 @@ internal static TokenDecryptionResult DecryptJwtToken(
                 var cryptoProviderFactory = validationParameters.CryptoProviderFactory ?? key.CryptoProviderFactory;
                 if (cryptoProviderFactory == null)
                 {
-                    if (LogHelper.IsEnabled(EventLogLevel.Warning))
-                        LogHelper.LogWarning(TokenLogMessages.IDX10607, key);
+                    // TODO: Move to CallContext
```

Comment:
will this happen in this PR or the next?

Reply:
Next. We are moving most of our logs out by providing all the validated data as part of the result.
Need to figure it we still need to provide this information through the CallContext or if that can be dropped.